src/cider/exec/nextgen/Nextgen.cpp

```diff
@@ -28,7 +28,7 @@
 namespace cider::exec::nextgen {
 
 std::unique_ptr<context::CodegenContext> compile(
-    const RelAlgExecutionUnit& ra_exe_unit,
+    RelAlgExecutionUnit& ra_exe_unit,
```

Comment:
Q: why need to remove ```const``` here? Same for 
https://github.com/intel/BDTK/pull/277/files#diff-e19320dad5e5a53117976e0a305558dae44f37af90b5f7ddf23929825f5c2f57R33

Reply:
I used the RelAlgExecutionUnit in class InputAnalyzer, run the traverse function with its member variable such as join_quals, shared_target_exprs etc. and needs to read and write it in traverse logic
https://github.com/intel/BDTK/pull/277/files#diff-bd9f1527faedcddf56a8c76d328735d0934369763a3e50a4315fc346df8b63c9R109